### PR TITLE
Interview lab

### PR DESCRIPTION
**Pull request by kstifel**
Please take a look at my updated README.md file. Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +31,17 @@ Q2: What is the answer?
 ## Interview
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
-Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?<br>
+*_Kate_: I'm currently a requirements analyst and have always wanted to become a developer. I was trying to learn on my own but it was overwhelming. I thought this would be the perfect opportunity.
 
-Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?<br>
+*_Kate_: My favorite place so far has been NYC. I would love to travel to Italy at some point.
```
Comment (jschmersal-cscc) [inline README.md]: Could you add some more about your NYC trip? What was one or two of the highlights?

File: README.md [hunk 2]
```diff
@@ -31,13 +31,17 @@ Q2: What is the answer?
...
 
-Q3: Name your favorite member of [The Beatles](https://en.wikipedia.org/wiki/The_Beatles), and your favorite Beatles song.  You do know at least one Beatles song, right?  If you are a Beatles fan, feel free to give me your top 5 or so.
+Q3: Name your favorite member of [The Beatles](https://en.wikipedia.org/wiki/The_Beatles), and your favorite Beatles song.  You do know at least one Beatles song, right?  If you are a Beatles fan, feel free to give me your top 5 or so.<br>
+*_Kate_: Favorite member of The Beatles is John Lennon, favorite song is Let it Be (not a huge Beatles person as you can probably tell by these generic answers)
 
-Q4: Do you have any pets? Species/Name/Details please. [Murray](images/Murray.jpeg?raw) wants to know... (he's not as smart as that picture makes him seem). 
+Q4: Do you have any pets? Species/Name/Details please. [Murray](images/Murray.jpeg?raw) wants to know... (he's not as smart as that picture makes him seem).<br>
+*_Kate_: I just got my first pet 3 months ago. He's a dog named Bowie. He's a beagle jack russell mix. We (my husband Gary and I) adopted him from the Richland County Animal Shelter and he's the light of our life!
```
Comment (jschmersal-cscc) [inline README.md:46]: Nice.  Murray would approve!

**Review by jschmersal-cscc — CHANGES_REQUESTED** [review]
Nice start! Please add a bit more details to your interview per the embedded comment and update the pull request.

**Comment by jschmersal-cscc** [comment]
Looks good! This part of the lab is finished.  When part two is finished please submit the link to this PR and the PR for part 2.  If possible, a screenshot of both PRs is also good to see in blackboard for posterity (you don't need to capture the entire PR, just some of the conversation).

**Comment by kstifel** [comment]
Thank you Jeremy! I made some formatting updates before I saw your comment.

**Pull request closed without merging**